app/src/main/java/de/triology/cas/services/EtcdClientFactory.java

```diff
@@ -2,23 +2,19 @@
 
 import mousio.etcd4j.EtcdClient;
 import org.apache.commons.lang.StringUtils;
-import org.springframework.context.annotation.Bean;
-import org.springframework.stereotype.Component;
 
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileReader;
 import java.io.IOException;
 import java.net.URI;
 
-@Component
 public class EtcdClientFactory {
 
     private static final String DEFAULT_NOTE_MASTER_FILE = "/etc/ces/node_master";
 
-    @Bean
-    public EtcdClient createEtcdClient() {
-        return createEtcdClient(DEFAULT_NOTE_MASTER_FILE);
+    public EtcdClient createDefaultFactory() {
```

Comment:
Why did the name change from `createEtcdClient()` to `createDefaultFactory()`? The function returns a client and not a factory, so the name should probably stay as it is, I think.

Reply:
Changed to `createDefaultClient`. Done